### ISSUE TITLE
Some typescript type errors

### ISSUE THREAD

**Issue by Izrab** [armackey/nativescript-twilio-video#12]
I copied the same code from demo to my app, but there appear some type warnings, I cannot cloud build my project, I had to add the code after the app is built. 
There are actually two type errors, one saying RemoteVideo is not export in module twilio plugin, and the other error says cannot find name AVCaptureVideo.
I use the latest twilio plugin (1.1.6), also I use NS 6. 

![Annotation 2019-07-27 180646](https://user-images.githubusercontent.com/22328131/61996154-f9b2fe00-b045-11e9-90eb-08df52e9dd20.png)
![Annotation 2019-07-27 180709](https://user-images.githubusercontent.com/22328131/61996155-fb7cc180-b045-11e9-9c91-46de9585b1c5.png)

**Comment by armackey** [comment]
The first error. I'm not sure why TS is saying it's an error. If you're not using, don't import it for now.
The second error requires you to install tns-platform-declarations. I recommend you look it.

**Comment by Izrab** [comment]
@armackey , for the first error, I removed "RemoteVideo", because it is not used anyway. Is that okay and fine??

**Comment by armackey** [comment]
Yep. That's fine.

**Comment by Izrab** [comment]
@armackey I installed tns-platform-declarations package, but the red type error did not go away

**Comment by Izrab** [comment]
Either the red errors go away or I cannot build my app, because webpack errors rise.

**Comment by armackey** [comment]
did you actually look at the documentation for `tns-platform-declarations`?

**Comment by armackey** [comment]
documentation clearly states how to do this. a simple install isn't gonna cut it.

http://prntscr.com/okuuvg

**Comment by Izrab** [comment]
the docs are not clear, where do I create the file? In app directory or parent directory?

**Comment by Izrab** [comment]
Alright the red type errors went away.

**Comment by armackey** [comment]
...i'll emphasize one last time. be sure to actually read docs.

**Comment by Izrab** [comment]
You're right, I should focus and be attentive and be caring, and read manuals. Thanks a lot. Now I just have the switch camera issue. You can close this issue, as the red warnings are gone.
